### PR TITLE
Ajustes transversales páginas de asignación

### PR DESCRIPTION
**Pull request by fermarinsanchez**
We added all the settings requested in the ticket.

### CODE REVIEW

File: plugins/leemons-plugin-tasks/frontend/src/components/Inputs/ConditionalInput/index.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
-import { Switch, Stack, PageContainer } from '@bubbles-ui/components';
+import { Switch, Stack, PageContainer, Box } from '@bubbles-ui/components';
```
Comment (MIGUELez11) [inline plugins/leemons-plugin-tasks/frontend/src/components/Inputs/ConditionalInput/index.js:3]:
```suggestion
import { Switch, Stack, PageContainer } from '@bubbles-ui/components';
```

Remove the unused Box

File: plugins/leemons-plugin-assignables/frontend/src/components/Assignment/components/GroupPicker/GroupPicker.js [hunk 2]
```diff
@@ -19,7 +19,10 @@ const useGroupPickerStyles = createStyles((theme) => ({
     gap: theme.other.global.spacing.gap.lg,
   },
   content: {
-    marginLeft: theme.other.global.spacing.padding.xlg,
+    // marginLeft: theme.other.global.spacing.padding.xlg,
+  },
```
Comment (MIGUELez11) [inline plugins/leemons-plugin-assignables/frontend/src/components/Assignment/components/GroupPicker/GroupPicker.js:23]: If unused, don't leave it commented, also the class

**Review by MIGUELez11 — CHANGES_REQUESTED** [review]

File: plugins/leemons-plugin-tasks/frontend/src/components/Inputs/ConditionalInput/index.js (hunk 1 above)
Comment (fermarinsanchez) [reply plugins/leemons-plugin-tasks/frontend/src/components/Inputs/ConditionalInput/index.js:3]: done

File: plugins/leemons-plugin-assignables/frontend/src/components/Assignment/components/GroupPicker/GroupPicker.js (hunk 2 above)
Comment (fermarinsanchez) [reply plugins/leemons-plugin-assignables/frontend/src/components/Assignment/components/GroupPicker/GroupPicker.js:23]: done

**Pull request merged**